<!DOCTYPE html>
<!--[if IE 8 ]><html class="no-js oldie ie8" lang="en"> <![endif]-->
<!--[if IE 9 ]><html class="no-js oldie ie9" lang="en"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!--><html class="no-js" lang="en"> <!--<![endif]-->
<head>

   <!--- basic page needs
   ================================================== -->
   <meta charset="utf-8">
	<title>officialprofile</title>
	<meta name="description" content="Brief summary of my work">  
	<meta name="author" content="officialprofile">

   <!-- mobile specific metas
   ================================================== -->
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

 	<!-- CSS
   ================================================== -->
   <link rel="stylesheet" href="css/base.css">  
   <link rel="stylesheet" href="css/main.css">
   <link rel="stylesheet" href="css/vendor.css">     

   <!-- script
   ================================================== -->   
	<script src="js/modernizr.js"></script>
	<script src="js/pace.min.js"></script>

   <!-- favicons
	================================================== -->
	<link rel="icon" type="image/png" href="favicon.png">

</head>

<body id="top">

	<!-- header 
   ================================================== -->
   <header>   	
  
   </header> <!-- /header -->

   <!-- resume Section
   ================================================== -->
	<section id="resume" class="grey-section">

		<div class="row section-intro">
   		<div class="col-twelve">
<h1><a href='https://github.com/officialprofile'><i class="fa fa-github"></i></a></h1>
   			<h5>officialprofile.github.io</h5>
   			<h1>Brief summary of my work</h1>
<!-- 
   			<p class="lead">Bioinformatics, with the emphasis on genomics, is the focal point of my research and projects that I do after hours. I also apply machine learning, statistics, data mining, visualization and mathematical modeling, both within professional and personal undertakings.</p>-->
      
   		</div>   		
   	</div> <!-- /section-intro--> 

   	<div class="row resume-timeline">

   		<div class="col-twelve resume-header">

   			<h2>Research articles</h2>

   		</div> <!-- /resume-header -->

   		<div class="col-twelve">

   			<div class="timeline-wrap">

				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-stethoscope"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2024</h3>
	   					<p>Genetics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Conditional deletion of miR-204 and miR-211 in murine retinal pigment epithelium results in retinal degeneration</h4>
	   					<p>MicroRNAs (miRs) are short, evolutionarily conserved non-coding RNAs that canonically downregulate expression of target genes. The miR family composed of miR-204 and miR-211 is among the most highly expressed in the retinal pigment epithelium (RPE) in both mouse and human, and also retains high sequence identity. To assess the role of this miR family in the developed mouse eye, we generated two floxed conditional knockout mouse lines crossed to the RPE65-ERT2-Cre driver mouse line to perform an RPE-specific conditional knockout of this miR family in adult mice. After Cre-mediated deletion, we observed retinal structural changes by optical coherence tomography; dysfunction and loss of photoreceptors by retinal imaging; and retinal inflammation marked by subretinal infiltration of immune cells by imaging and immunostaining. Single-cell RNA sequencing of diseased RPE and retinas showed potential miR-regulated target genes, as well as changes in non-coding RNAs in the RPE, rod photoreceptors, and Müller glia. This work thus highlights the role of miR-204 and miR-211 in maintaining RPE function and how the loss of miRs in the RPE exerts effects on the neural retina, leading to inflammation and retinal degeneration.</p>
	   					<p>Journal of Biological Chemistry  <b>120</b>(50). DOI: 10.1016/j.jbc.2024.107344</p>			
	   					
	   				</div>
				
				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-stethoscope"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2023</h3>
	   					<p>Genetics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Distinct mouse models of Stargardt disease display differences in pharmacological targeting of ceramides and inflammatory responses</h4>
	   					<p>Mutations in many visual cycle enzymes in photoreceptors and retinal pigment epithelium (RPE) cells can lead to the chronic accumulation of toxic retinoid byproducts, which poison photoreceptors and the underlying RPE if left unchecked. Without a functional ATP-binding cassette, sub-family A, member 4 (ABCA4), there is an elevation of all-trans-retinal and prolonged buildup of all-trans-retinal adducts, resulting in a retinal degenerative disease known as Stargardt-1 disease. Even in this monogenic disorder, there is significant heterogeneity in the time to onset of symptoms among patients. Using a combination of molecular techniques, we studied Abca4 knockout (simulating human noncoding disease variants) and Abca4 knock-in mice (simulating human misfolded, catalytically inactive protein variants), which serve as models for Stargardt-1 disease. We compared the two strains to ascertain whether they exhibit differential responses to agents that affect cytokine signaling and/or ceramide metabolism, as alterations in either of these pathways can exacerbate retinal degenerative phenotypes. We found different degrees of responsiveness to maraviroc, a known immunomodulatory CCR5 antagonist, and to the ceramide-lowering agent AdipoRon, an agonist of the ADIPOR1 and ADIPOR2 receptors. The two strains also display different degrees of transcriptional deviation from matched WT controls. Our phenotypic comparison of the two distinct Abca4 mutant-mouse models sheds light on potential therapeutic avenues previously unexplored in the treatment of Stargardt disease and provides a surrogate assay for assessing the effectiveness for genome editing.</p>
	   					<p>Proceedings of the National Academy of Sciences  <b>120</b>(50). DOI: 10.1073/pnas.2314698120</p>			
	   					
	   				</div>
				
				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-stethoscope"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2023</h3>
	   					<p>Medicine</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Comparison of Diffusion Kurtosis Imaging and Standard Mono-Exponential Apparent Diffusion Coefficient in Diagnosis of Significant Prostate Cancer — A Correlation with Gleason Score Assessed on Whole-Mount Histopathology Specimens</h4>
	   					<p>The study was undertaken to compare the diagnostic performance of diffusion kurtosis imaging (DKI) with the standard monoexponential (ME) apparent diffusion coefficient (ADC) model in the detection of significant prostate cancer (PCa), using whole-mount histopathology of radical prostatectomy specimens as a reference standard. 155 patients with prostate cancer had undergone multiparametric magnetic resonance imaging (mpMRI) at 3T before prostatectomy. Quantitative diffusion parameters—the apparent diffusion coefficient corrected for non-Gaussian behavior (Dapp), kurtosis (K), ADC1200, and ADC2000 were correlated with Gleason score and compared between cancerous and benign tissue and between GS ≤ 3 + 3 and GS ≥ 3 + 4 tumors. The mean values of all diffusion parameters (Dapp, K, ADC1200, ADC2000) were significantly different both between malignant and benign tissue and between GS ≤ 3 + 3 and GS ≥ 3 + 4 tumors. Although the kurtosis model was better fitted to DWI data, the diagnostic performance in receiver operating characteristic (ROC) analysis of DKI and the standard ADC model in the detection of significant PCa was similar in the peripheral zone (PZ) and in peripheral and transitional zones (TZ) together. In conclusion, our study was not able to demonstrate a clear superiority of the kurtosis model over standard ADC in the diagnosis of significant PCa in PZ and in both zones combined.</p>
	   					<p>Diagnostics <b>13</b>(2):173. DOI: 10.3390/diagnostics13020173.</p>			
	   					
	   				</div>

   				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-laptop"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2022</h3>
	   					<p>Machine learning</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Convolutional Neural Networks for C. Elegans Muscle Age Classification Using Only 
Self-Learned Features</h4>
	   					<p>Nematodes <i>Caenorhabditis elegans</i> (<i>C. elegans</i>) have been used as model organisms in a wide variety of biological studies, especially oriented towards better understanding aging and age-associated diseases. This paper concerns the automation of <i>C. elegans</i> imagery analysis to classify the muscle age of nematodes based on the known and well established IICBU dataset. Unlike many modern classification methods, the proposed approach utilizes Deep Learning techniques, specifically Convolutional Neural Networks (CNNs) to solve the problem and achieve high classification accuracy by focusing on non-handcrafted self-learned features. Various networks known from ImageNet Large Scale Visual Recognition Challenge (ILSVRC) were investigated and adapted for the <i>C. elegans</i> muscle aging dataset via transfer learning and data augmentation techniques. The proposed approach of unfreezing different numbers of convolutional layers in the feature extraction stage, and introducing different structures of newly trained fully connected layers in the classification stage, allowed for better fine-tuning of the selected networks. Adjusted CNNs featured in this paper have been compared with other state-of-art methods. In antiaging drug research, the proposed CNN would be a very fast and effective age determination method, which would lead to reductions in time and costs of laboratory research.</p>
	   					<p>Journal of Telecommunications and Information Technology 4/2022. DOI: 10.26636/jtit.2022.165322.</p>			
	   					
	   				</div>


   				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-stethoscope"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2021</h3>
	   					<p>Medicine</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>The Role of the Trabecular Bone Score in the Assessment of Osteoarticular Disorders in Patients with HFE-Hemochromatosis: A Single-Center Study from Poland</h4>
	   					<p>Type 1 hereditary hemochromatosis (HH) is an autosomal, recessive genetic entity with systemic iron overload. Iron homeostasis disorders develop as a result of HFE gene mutations, which are associated with hepcidin arthropathy or osteoporosis and may cause permanent disability in HH patients despite a properly conducted treatment with phlebotomies. In this study, selected parameters of calcium and phosphate metabolism were analyzed in combination with the assessment of bone mineral density (BMD) disorders in patients from northern Poland with clinically overt HFE-HH. BMD was determined by a dual-energy X-ray absorptiometry (DXA) test with the use of the trabecular bone score (TBS) function.</p>
	   					<p> Genes <b>12</b>(9):1304. DOI: 10.3390/genes12091304.</p>			
	   					
	   				</div>


   				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-laptop"></i>
	   				</div>

<div class="timeline-header">
	   					<h3>2021</h3>
	   					<p>Bioinformatics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Applications of 2D and 3D-Dynamic Representations of DNA/RNA Sequences for a description of genome sequences of viruses</h4>
	   					<p>The aim of the studies is to show that graphical bioinformatics methods are good tools for the description of genome sequences of viruses. A new approach to the identification of unknown virus strains is proposed. </p>
	   					<p> Combinatorial Chemistry & High Throughput Screening <b>72</b>(2).<br/> DOI: 10.2174/1386207324666210804120454.</p>			
	   					
	   				</div>
	   				
	   			<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-stethoscope"></i>
	   				</div>
	   				<div class="timeline-header">
	   					<h3>2021</h3>
	   					<p>Medicine</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Low incidence of focal lesions in the thyroid glands  of patients with hereditary haemochromatosis  — a single-centre study from Poland</h4>
	   					<p>Hereditary haemochromatosis (HH) is a disease characterised by the excessive absorption of iron and its deposition in various organs. Late complications of this disease include cirrhosis, hepatocellular carcinoma, and endocrine disorders. Data from the literature on thyroid disorders in patients with HH are inconsistent and ambiguous, and no research has been done to determine the relationship between excessive accumulation of iron and the thyroid morphology. Therefore, the aim of this study was to characterise thyroid function and ultrasound images in patients with clinically overt hereditary haemochromatosis.</p>
	   					<p>Endokrynologia Polska <b>72</b>(2). DOI: 0.5603/EP.a2021.0008.</p>			
	   					
	   				</div>

	   			</div> <!-- /timeline-block -->

	   			<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-laptop"></i>
	   				</div>

	   				<div class="timeline-header">
	   					<h3>2019</h3>
	   					<p>Bioinformatics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>Dynamic Representations of Biological Sequences</h4>
	   					<p>Methods of bioinformatics in which the biological sequences (DNA, RNA, protein) are represented by sets of material points in 2D, 3D, or 20D space, and described by values analogous to the ones used in the dynamics, as e.g. moments of inertia, are reviewed. A new application of the 3D method, called by us 3D-Dynamic Representation of DNA/RNA Sequences is proposed. It is shown that the method is useful for a description of complete genome sequences of dengue virus.</p>
	   					
	   					<p>MATCH Communications in Mathematical and in Computer Chemistry <b>82</b>(1):205-218. ISSN 0340 - 6253.</p>
	   				</div>

	   			</div> <!-- /timeline-block -->


	   			<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-laptop"></i>
	   				</div>

	   				<div class="timeline-header">
	   					<h3>2018</h3>
	   					<p>Bioinformatics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>An Application of the 2D-Dynamic Representation of DNA/RNA Sequences to the Prediction of Influenza A Virus Subtypes</h4>
	   					<p>A new theoretical method for the virus identifcation has been proposed. The 2D-Dynamic Representation of DNA/RNA Sequences has been applied to the prediction of influenza A virus subtypes. We have shown that the method can be successfully combined with novel supervised machine learning algorithms, such as C5.0. The descriptors of the 2D-Dynamic Representation of DNA/RNA Sequences have been evaluated. High mean accuracy of predicting the subtype of the influenza A virus has been obtained (over 90% of correct predictions). As a consequence, the combination of the machine learning algorithms and the 2D-Dynamic Representation of DNA/RNA Sequences has been shown to constitute a simple and accurate tool for the classifcation of unidentifed virus strains.</p>
	   					
	   					<p>MATCH Communications in Mathematical and in Computer Chemistry <b>80</b>(2):295-310. ISSN: 0340-6253.</p>
	   				</div>

	   			</div> <!-- /timeline-block -->

	   			<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-laptop"></i>
	   				</div>

	   				<div class="timeline-header">
	   					<h3>2017</h3>
	   					<p>Bioinformatics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>2D-dynamic representation of DNA/RNA sequences as a characterization tool of the zika virus genome</h4>
	   					<p>2D-dynamic representation of DNA/RNA sequences has been applied for the characterization of the complete genome sequence of Zika virus. Graphically, the 2D-dynamic graphs evolve with time. Numerically, applying descriptors related to the 2D-dynamic graphs, correct classification of the sequences has been obtained. These descriptors have been shown to give an adequate characteristics of the Zika virus genome. The classification diagrams form a new mathematical description of the evolution of the genome sequence of the Zika virus.</p>
	   					
	   					<p>MATCH Communications in Mathematical and in Computer Chemistry <b>77</b>(2):321-332. ISSN: 0340-6253.</p>
	   				</div>

	   			</div> <!-- /timeline-block -->

   			</div> <!-- /timeline-wrap -->   			

   		</div> <!-- /col-twelve -->
   		
   	</div> <!-- /resume-timeline -->
   	
   	<div class="row resume-timeline">

   		<div class="col-twelve resume-header">

   			<h2>Popular science</h2>

   		</div> <!-- /resume-header -->

   		<div class="col-twelve">

   			<div class="timeline-wrap">

   				<div class="timeline-block">

	   				<div class="timeline-ico">
	   					<i class="fa fa-newspaper-o"></i>
	   				</div>

	   				<div class="timeline-header">
	   					<h3>2021</h3>
	   					<p>Mathematics</p>
	   				</div>

	   				<div class="timeline-content">
	   					<h4>O rzucaniu monetą (Tossing a coin)</h4>
	   					<p>The repetition of a simple random experiment, such as the toss of a coin, can lead to extremely complex phenomena. Genetic diseases or human intelligence are just a few examples that are discussed in this article.</p>
	   					<p>ForuM. Biuletyn Mensy Polskiej 219:6-10. ISSN: 1425-0136.</p>
	   				</div>

	   			</div> <!-- /timeline-block -->

   			</div> <!-- /timeline-wrap -->   			

   		</div> <!-- /col-twelve -->
   		
   	</div> <!-- /resume-timeline -->
		
	</section> <!-- /features -->





   <div id="preloader"> 
    	<div id="loader"></div>
   </div> 

   <!-- Java Script
   ================================================== --> 
   <script src="js/jquery-2.1.3.min.js"></script>
   <script src="js/plugins.js"></script>
   <script src="js/main.js"></script>

</body>

</html>
